dev/com.ibm.ws.security.spnego_fat.1/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/SPNEGOTokenHelperFVTWeb.war/src"/>
```

Comment:
No test-applications for this project, so this should be removed.

Reply:
Yes. This can be removed. This app was already in the fat.common project, so I just use it from there.